### PR TITLE
Remove timeline preview link from nav panel when not signed-in

### PR DESCRIPTION
**Pull request by ykzts**

### CODE REVIEW

**Review by adisagar2003 — APPROVED** [review]

File: app/javascript/mastodon/components/server_banner.js [hunk 1]
```diff
@@ -62,23 +62,25 @@ class ServerBanner extends React.PureComponent {
...
-              </>
-            ) : (
-              <>
-                <strong className='server-banner__number'><ShortNumber value={server.getIn(['usage', 'users', 'active_month'])} /></strong>
-                <br />
-                <span className='server-banner__number-label' title={intl.formatMessage(messages.aboutActiveUsers)}><FormattedMessage id='server_banner.active_users' defaultMessage='active users' /></span>
-              </>
-            )}
-          </div>
+          {activityApiEnabled && (
```
Comment (Gargron) [inline app/javascript/mastodon/components/server_banner.js:65]: This number is not connected to this API. Why this change?

Comment (ykzts) [reply app/javascript/mastodon/components/server_banner.js:65]: Oops, I misunderstood as I was displaying statistics. I will revert this change.

**Review by Gargron — APPROVED** [review]
